install.sh

```diff
@@ -17,11 +17,18 @@ then
         exit 1
     fi
 else
-    echo "Node detected"
+    DETECTED="$(node -v)"
+    echo $DETECTED 
+    MINIMAL="v16.10.0"
+    SORTED=($(for VER in "$DETECTED" "$MINIMAL"; do echo "$VER"; done | sort -V))
+    if [ "${SORTED[0]}" = "$DETECTED" ]; then
+        echo "Minimal supported version is" $MINIMAL
+        exit 1
+    fi
```

Comment:
If we already now that the user has NodeJS installed I would do this.

```shell
MINIMAL="v16.10.0"
DETECTED=`node -v`

RESULT=`node -e "console.log(\"$DETECTED\".replace('v', '').localeCompare(\"$MINIMAL\".replace('v', ''), undefined, { numeric: true }))"`

if [ $RESULT -lt 0 ]; then
        echo "Minimal supported version is" $MINIMAL
        exit 1
fi

Reply:
👍 but I'll rather avoid `localCompare`